chore(docs): added prompt to update tools.rst in build-docs.sh

Doug-AWS opened the pull request:
----

### Pull Request Checklist

Please check all boxes (including N/A items)

#### Testing

- [ ] Unit test and/or integration test added
- [ ] __Toolkit change?:__ [integration
tests](https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk/integ-tests/test.sh)
manually executed (paste output to the PR description)
- [ ] __Init template change?:__ coordinated update of integration tests
(currently maintained in a private repo).

#### Documentation

- [x] __README__: README and/or documentation topic updated
- [ ] __jsdocs__: All public APIs documented

### Title and description

- [x] __Change type__: Title is prefixed with change type:
  * `fix(module): <title>` bug fix (_patch_)
  * `feat(module): <title>` feature/capability (_minor_)
  * `chore(module): <title> ` won't appear in changelog
  * `build(module): <title>` won't appear in changelog
- [x] __Title format__: Title uses lower case and doesn't end with a period
- [ ] __Breaking change?__: Last paragraph of description is: `BREAKING
  CHANGE: <describe exactly what changed and how to achieve similar behavior +
  link to documentation/gist/issue if more details are required>`
- [ ] __References__: Indicate issues fixed via: `Fixes #xxx` or `Closes #xxx`

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

eladb commented:
It's part of this repo, so you should be able to execute it as:

    ../packages/aws-cdk/bin/cdk

This is the location relative to the `docs` directory.

Doug-AWS commented:
../packages/aws-cdk/bin/cdk is just a wrapper for ../packages/aws-cdk/bin/cdk.js, which does not exist. I had to add the following BEFORE calling ../packages/aws-cdk/bin/cdk:

tsc # in bin/

Gave me 150+ errors, so I added tsconfig.json to the bin/ directory:

{
    "compilerOptions": {
        "target": "es2018",
        "module": "commonjs"
    }
}

I got 23 errors, such as:

cdk.ts(4,24): error TS2307: Cannot find module '@aws-cdk/cx-api'.

So I added node_modules/.bin (in the cdk root and cdk/packages/aws-cdk) to the path, but I still get the same errors. Ideas? I've saved my build output to build.txt.

eladb commented on docs/build-docs.sh:
```diff
@@ -82,4 +82,29 @@ fi
+PATH=$PWD/../node_modules/.bin:$PATH
+echo "PATH:"
+echo $PATH
+if [ ! -d ../packages/aws-cdk/bin/cdk.js ]; then
```
Sadly it's more complicated that that. The toolkit has dependencies that must be built first. Here are your options:

1. You could use the [`buildup`](https://github.com/awslabs/aws-cdk/blob/master/scripts/buildup) script if you like
2. Just print an error saying "you must first build aws-cdk in order for this to work"
3. Put placeholder text instead of the usage string in case the toolkit is not available (in the real build it will be available).

Doug-AWS commented:
I picked door number 2: I barf out a message that you need to build the CDK and then run update-tools.sh.

RomainMuller commented on docs/build.txt:
```diff
@@ -0,0 +1,45 @@
+Staging Sphinx doc site under .staging
```
I don't reckon this should be committed.

rix0rrr commented:
No longer relevant in light of docs movement, right? I will close for now.
